python/core/auto_generated/layertree/qgslayertreenode.sip.in

```diff
@@ -208,6 +208,13 @@ Returns a list of any checked layers which belong to this node or its
 children.
 
 .. versionadded:: 3.0
+%End
+
+    int depth() const;
+%Docstring
+Returns the depth of this node, i.e. the number of it's ancestors
```

Comment:
HMm - sipify test should be rejecting this as out-of-sync...

Reply:
Ahh I see, I don't think you've pushed your "it's" fix yet?

Reply:
I missed that one, should be ok now